docs/csharp/language-reference/compiler-messages/cs0173.md

```diff
@@ -10,28 +10,71 @@ ms.assetid: eb1797ad-bf62-4e2b-8922-bef4aff36954
 ---
 # Compiler Error CS0173
 
-Type of conditional expression cannot be determined because there is no implicit conversion between 'class1' and 'class2'
+Type of conditional expression cannot be determined because there is no implicit conversion between 'type1' and 'type2'
```

Comment:
Is it okay for accessibility (screen readers) that these don't end in punctuation?